forge-game/src/main/java/forge/game/staticability/StaticAbility.java

```diff
@@ -583,6 +583,13 @@ public int compareTo(StaticAbility arg0) {
         .result();
     }
 
+    public long getTimestamp() {
```

Comment:
imo already refactor `Card.suspectedEffect` this way here
so we know if it works before adding extra code for it